src/components/IntegrationTest/IntegrationTestForm/IntegrationTestSection.tsx

```diff
@@ -183,23 +150,13 @@ const IntegrationTestSection: React.FC<React.PropsWithChildren<Props>> = ({ isIn
           data-test="git-path-repo"
           required
         />
-        <DropdownField
```

Comment:
Do we not support any kind of ephemeral environments for integration tests now?

Reply:
As discussed on slack, Right now there is no UI change required in place of Environments for integration tests.